flow2ml/Flow.py

```diff
@@ -202,3 +204,23 @@ def getDataset(self,img_dimensions,train_val_split):
     self.update_progress( 100/100.0,"Created Datasets" ) 
 
     return self.dataset
+
+  def deployTensorflowModels(self,model):
```

Comment:
You passed again only 1 parameter here. Please include self, conversions, model as inputs here